### PR TITLE
Add endoplasmic reticulum API for morphio.mut

### PR DESCRIPTION
**Pull request by wizmer**

### CODE REVIEW

**Comment by asanin-epfl** [comment]
I've just started reviewing and you have fixed all of my notes already XD

File: tests/test_0_API.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def methods(cls):
         return set(method for method in dir(cls) if not method[:2] == '__')
 
     only_in_immut = {'section_types', 'diameters', 'perimeters', 'points', 'section_offsets',
-                     'as_mutable', 'endoplasmic_reticulum'}
+                     'as_mutable'}
```
Comment (asanin-epfl) [inline tests/test_0_API.py]: Should not we add `morphio.mut.EndoplasmicReticulum` to `classes` above?

File: README.md [hunk 2]
```diff
@@ -402,6 +403,52 @@ for mitochondrial_section in morpho.mitochondria.root_sections:
...
+This dataset represents the index of the neuronal section that the reticulum lies in.
+
+- volume:
+A 1 column data set composed of floating point numbers representing the volume of the part of the reticulum lying in section referenced by the corresponding row in the section_index data set.
+
+- surface_area:
+Similar to the volume data set, this data set represents the surface area of the reticulum in each section in the section_index data set.
+
+- filament_count:
+This 1 column data set is composed of integers that represent the number of filaments in the segment of the reticulum lying in the section referenced by the corresponding row in the section_index data set.
```
[the remark this answers is not in the document]
Comment (wizmer) [reply README.md]: Actually I did not wrote it myself. I took it from the morphology documentation page :D

File: README.md [hunk 3]
```diff
@@ -402,6 +403,52 @@ for mitochondrial_section in morpho.mitochondria.root_sections:
...
+
+
+#### Reading endoplasmic reticula from H5 files
```
[the remark this answers is not in the document]
Comment (wizmer) [reply README.md]: reticula is the plural of reticulum.

File: README.md [hunk 4]
```diff
@@ -402,6 +403,52 @@ for mitochondrial_section in morpho.mitochondria.root_sections:
...
 ```
+### Endoplasmic reticulum
+
+Endoplasmic reticulum can also be stored and written to H5 file.
+The specification is part of the [H5 documentation](https://bbpteam.epfl.ch/documentation/projects/Morphology%20Documentation/latest/h5v1.html)
+There is one endoplasmic reticulum object per morphology. It contains 4 attributes. Each attribute is a array and each line
+refer to the value of the attribute for a specific neuronal section.
+
+- section_index:
+This dataset represents the index of the neuronal section that the reticulum lies in.
```
[the remark this answers is not in the document]
Comment (wizmer) [reply README.md]: I have tried something different:
Each row of this dataset represents the index of a neuronal section. Each row of the other properties (eg. volume) refer to the part of the reticulum
present in the corresponding section for each row.

[the remark this answers is not in the document]
Comment (asanin-epfl) [reply README.md]: Definitely Tolstoy, no less :writing_hand:

**Comment by wizmer** [comment]
The changes are approved but not the review. Is it a new GitHub thing ???

**Comment by asanin-epfl** [comment]
Probably I don't have sufficient rights.

**Pull request merged**